### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by RossKen**
### Type of PR

- [ ] BUG
- [ ] FEAT
- [ ] MAINT
- [x] DOC

### PR Checklist

- [ ] Added documentation for changes
- [ ] Added feature to example notebooks or tutorial (if appropriate)
- [ ] Added tests (if appropriate)
- [ ] Updated CHANGELOG.md (if appropriate)
- [ ] Made changes based off the latest version of Splink
- [ ] Run the [linter](https://moj-analytical-services.github.io/splink/dev_guides/changing_splink/lint.html)

### CODE REVIEW

File: docs/demos/examples/duckdb/sensitivity_analysis.ipynb
Comment (ADBond) [inline docs/demos/examples/duckdb/sensitivity_analysis.ipynb:1]: was this file meant to be added in this PR?

File: docs/dev_guides/index.md [hunk 1]
```diff
@@ -27,6 +27,6 @@ Splink is quite a large, complex codebase. The guides in this section lay out so
 * [Transpilation using sqlglot](./transpilation.md) - demonstrates how Splink translates SQL in order to be compatible with multiple SQL engines using the sqlglot package.
 * [Performance and caching](./caching.md) - demonstrates how pipelining and caching is used to make Splink run more efficiently.
 * [Comparison and Comparison Level Libraries](./comparisons/new_library_comparisons_and_levels.md) - demonstrates how `Comparison` Library and `ComparisonLevel` Library functions are structured within Splink, including how to add new functions and edit existing functions.
-* [Charts](./charts.ipynb) - demonstrates how charts are built in Splink, including how to add new charts and edit existing charts.
+* [Charts](./charts/understanding_and_editing_charts.md.ipynb) - demonstrates how charts are built in Splink, including how to add new charts and edit existing charts.
```
Comment (ADBond) [inline docs/dev_guides/index.md:30]: think this might have an extra suffix

**Review by ADBond — APPROVED** [review]
Haven't checked in all detail but looks sensible - can mop up any extras later if needs be.
Just a couple of things but happy for you to go ahead and merge when you're ready

File: docs/demos/examples/duckdb/sensitivity_analysis.ipynb
Comment (RossKen) [reply docs/demos/examples/duckdb/sensitivity_analysis.ipynb:1]: no - am removing

File: docs/dev_guides/index.md (hunk 1 above)
Comment (RossKen) [reply docs/dev_guides/index.md]: good spot - that was a stupid

**Comment by RossKen** [comment]
Thanks for the review @ADBond!

**Pull request merged**